app/templates/sites.hbs

```diff
@@ -1,15 +1,23 @@
 <div class="row">
 
 <div class="col-xs-4">
-	
-{{#each model as |company|}}
-	<h4>
-		{{link-to company.name "sites.companies.company" company data-role="company-link"}}
-	</h4>
-	{{#each company.networkSites as |networkSite|}}
-		{{network-site-listing networkSite=networkSite}}
-	{{/each}}
-{{/each}}
+
+  {{#link-to
+      "sites.companies.new"
+      data-role="new-company-btn"
```

Comment:
Consider using `[data-test-selector]` here instead of `[data-role]` https://dockyard.com/blog/2015/09/25/ember-best-practices-acceptance-tests